src/python/pants/option/global_options.py

```diff
@@ -388,9 +391,17 @@ def register_bootstrap_options(cls, register):
 
     # This should eventually deprecate the RunTracker worker count, which is used for legacy cache
     # lookups via CacheSetup in TaskBase.
-    register('--process-execution-parallelism', type=int, default=multiprocessing.cpu_count(),
+    register('--process-execution-parallelism', type=int, dest='local_execution_parallelism',
+             # FIXME (hrfuller) what is the correct deprecation start version?
```

Comment:
I think +0.2.0.dev2 currently; ie 1.20.0.dev2

Reply:
Can you explain the deprecation process a bit more? I had assumed this change would ship with the next release, 1.19.x, is the 1.19.x release point already cut?

Reply:
Its all defined here: https://www.pantsbuild.org/deprecation_policy.html
The doc does appear out of date on the +.dev2 part though. That I think was only agreed on in slack and never formalized.

Reply:
Also, the args you want here are the `removal_version` and `removal_hint`. So `removal_version='1.20.0.dev2', removal_hint='Use that other thing instead.'`.

`deprecation_start_version` is when the deprecation begins, and is generally not necessary unless you want a deprecation to start sometime in the future.

Reply:
FWIW I'd be surprised if anyone were actually using this flag, and just deleting it probably wouldn't be the end of the world.

Reply:
I kinda figured the same, but then again, procedures.